cmd/kube-apiserver/app/server.go

```diff
@@ -240,6 +241,8 @@ func (s *APIServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.KubeletConfig.CertFile, "kubelet-client-certificate", s.KubeletConfig.CertFile, "Path to a client cert file for TLS.")
 	fs.StringVar(&s.KubeletConfig.KeyFile, "kubelet-client-key", s.KubeletConfig.KeyFile, "Path to a client key file for TLS.")
 	fs.StringVar(&s.KubeletConfig.CAFile, "kubelet-certificate-authority", s.KubeletConfig.CAFile, "Path to a cert. file for the certificate authority.")
+	// TODO: delete this flag when we enable global validation by default. See #14126
+	fs.BoolVar(&rest.CommonUpdateValidation, "common-update-validation", false, "If true, validation of common fields, like UID, will be enabled for all update (PUT) requests.")
```

Comment:
I would prefer to make this flag a "be nice to non-compliant clients" flag. We should always do the validation. The question is whether we reject bad requests or fix them.